docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -261,6 +261,9 @@ To generate key input events:
        /*
           Since Tizen 4.0, you can create an input generator with a given name (in this example, "Local Device"):
           inputgen = efl_util_input_initialize_generator_with_name(EFL_UTIL_INPUT_DEVTYPE_KEYBOARD, "Local_Device");
+
+          Since Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):
```

Comment:
Suggestion:
From, Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):
Is it "input generator" or "input device"?